recipes/_install-windows.rb

```diff
@@ -19,6 +19,33 @@
 
 dd_agent_version = Chef::Datadog.agent_version(node)
 
+# todo: make this dynamic
+BIN_PATH = 'C:/Program Files/Datadog/Datadog Agent/bin/agent'
+
+def fetch_current_version
+  return nil unless File.exist?(BIN_PATH)
+
+  agent_status = %x{"#{BIN_PATH}" status}
+  if match_data = agent_status.match(/^Agent \(v(.*)\)/)
+    Gem::Version.new(match_data[1])
+  end
+end
+
+def requested_agent_version
+  if version = Chef::Datadog.agent_version(node)
+    Gem::Version.new(version)
+  end
+end
+
+def must_reinstall?
+  current_version = fetch_current_version
+
+  return false unless current_version && requested_agent_version
+  return false unless requested_agent_version < current_version
+
+  true
```

Comment:
```suggestion
  requested_agent_version < current_version
```
And remove the second `return false unless ...`.